bin/tape

```diff
@@ -41,6 +41,18 @@ if (typeof opts.ignore === 'string') {
     var matcher = ignore.createMatcher(ignoreStr);
 }
 
+function supportsEsm() {
+    var version = process.version.match(/^v(\d+)\.(\d+)\.\d+$/);
+    var major = parseInt(version[1], 10);
+    var minor = parseInt(version[2], 10);
+
+    return major >= 14 || major === 13 && minor >= 2 || major === 12 && minor >= 17;
```

Comment:
if we're trying to truly detect ESM, we'd have to determine if `import()` is a parsing error - but `import()` parses in node v13.2-v13.6 even though it can't work - `import()` always rejects in these versions.

If we're trying to check where it can work, then let's use https://www.npmjs.com/package/has-package-exports which already does this check properly.

Reply:
i guess if testing for parsing errors can give false positives the safest would be to just use the second option... however it seems package exports (conditional) were unflagged in 12.17 and 13.7, while esm was unflagged in 12.17 and 13.2. just tested and `import()` works for me in 13.2.0, so this would lead to false negatives for node 13.2 - 13.6.

Reply:
Works how? In node v13.2, when i `import()` an mjs file, i get "Error: Cannot find module".

Reply:
This is most strange. I used nvm to install versions 13.2.0 - 13.6.0 and they all seem to work. My rudimentary test involved first creating the following files:

```js
// module-a.mjs
console.log(process.version)
import('./module-b.mjs').then(console.log)

// module-b.mjs
export default 1337
```

Running `$ node ./module-a.mjs` with node 13.2.0 prints
```
(node:481507) ExperimentalWarning: The ESM module loader is experimental.
v13.2.0
[Module] { default: 1337 }
```

Reply:
hmm, you're right. maybe it just doesn't work in the repl.

in that case, we do want to include v13.2. For now, let's keep your check, but i'll make a proper package to detect it and we can use that once it's available.

Reply:
repl in earlier versions resolved imports relative to the parent directory.